agent/engine/docker_task_engine.go

```diff
@@ -1138,7 +1143,27 @@ func (engine *DockerTaskEngine) AddTask(task *apitask.Task) {
 		engine.emitTaskEvent(task, err.Error())
 		return
 	}
-
+	if task.IsEBSTaskAttachEnabled() {
+		if ebsCsiDaemonManager, ok := engine.daemonManagers["ebs-csi-driver"]; ok {
```

Comment:
todo should the capability be a const?  This needs to be tied to the import dir name.  This should all be config.